Fix issue DPTP-3440 Add server-side apply support (#3417)
openshift/ci-tools: cmd/applyconfig/README.md

   resource that would be placed in that namespace are not yet present in the cluster (in a normal dry run, applying the
   resource to a namespace that does not exist in the cluster would fail). It does so by bookkeeping which namespaces
   would be applied if not in dry run, and creating them temporarily if they do not exist.

> 7. Server-side apply is always enabled for file with name starts with `_SS`

Reviewer: ```suggestion
7. Add `--server-side` to the `apply` command if a filename starts with `server-side_`
```

The file would be ignored if its filename starts with `_`.
